app/templates/partials/delete_task_modals.plush.html

```diff
@@ -0,0 +1,21 @@
+<div class="modal fade" id="deleteTaskModal" tabindex="-1" role="dialog" aria-labelledby="deleteTaskModalLabel" aria-hidden="true">
```

Comment:
We have as many id="deleteTaskModal" as there are todos. That's because we are in a loop.
HTML ids are unique, so we should avoid this. What about adding an index?
id="deleteTaskModal-<%=index%>"
If you do this, recall modifying the modal target
data-target="#deleteTaskModal"
to match the desired modal.
e.g: data-target="#deleteTaskModal-<%=index%>"